### PR TITLE
Added product-summary-sku-name

### PR DESCRIPTION
**Pull request by wender**
#### What problem is this solving?
The client wants to add the Product's SKU name to the shelf, current solution is to load the product-name component set the property to showSku and via CSS change the aspects of it generating extra useless information on the page.

#### How to test it?
Add `product-summary-sku-name` to the `product-summary.shelf`

```diff
"product-summary.shelf": {
    "children": [
      "product-summary-add-to-list-button",
      "stack-layout#shelf-image-flag",
      "product-summary-name",
+    "product-summary-sku-name",
      "product-rating-inline",
      "product-location-availability",
      "product-summary-price",
      "product-summary-sku-selector",
      "add-to-cart-button"
    ],
```

[Workspace](https://wender--eriksbikeshop.myvtex.com/Cycling/Bicycles/Road)

#### Screenshots or example usage:
Screen Shot 2020-09-22 at 16 51 15

### CODE REVIEW

File: .eslintrc [hunk 1]
```diff
@@ -5,6 +5,5 @@
     "node": true,
     "es6": true,
     "jest": true
-  },
-  "parser": "babel-eslint"
```
Comment (kaisermann) [inline .eslintrc:9]: Why did you remove this?

File: react/ProductSummarySKUName.tsx [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+import { path } from 'ramda'
+import { useCssHandles } from 'vtex.css-handles'
+import { useProductSummary } from 'vtex.product-summary-context/ProductSummaryContext'
+
+const CSS_HANDLES = ['skuNameContainer']
+
+const ProductSummarySKUName = () => {
+  const { product } = useProductSummary()
+  const handles = useCssHandles(CSS_HANDLES)
+  const skuName: string = path(['sku', 'name'], product) ?? ''
```
Comment (kaisermann) [inline react/ProductSummarySKUName.tsx:13]: You can replace `path` with the optional operator `?.`

```suggestion
  const skuName: string = product?.sku?.name ?? ''
```

File: react/ProductSummarySKUName.tsx [hunk 3]
```diff
@@ -0,0 +1,29 @@
+/* eslint-disable no-console */
+import React from 'react'
+// eslint-disable-next-line no-restricted-imports
+import { path } from 'ramda'
```
Comment (kaisermann) [inline react/ProductSummarySKUName.tsx:4]:
```suggestion
```

File: react/ProductSummarySKUName.tsx [hunk 4]
```diff
@@ -0,0 +1,29 @@
+/* eslint-disable no-console */
```
Comment (kaisermann) [inline react/ProductSummarySKUName.tsx:1]:
```suggestion
/* eslint-disable no-console */
```

**Review by kaisermann — CHANGES_REQUESTED** [review]

File: .eslintrc (hunk 1 above)
Comment (wender) [reply .eslintrc]: `vtex setup` did

File: react/ProductSummarySKUName.tsx [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
```
Comment (kaisermann) [inline react/ProductSummarySKUName.tsx:1]: Remove this please 🙏

**Review by kaisermann — APPROVED** [review]

File: react/ProductSummarySKUName.tsx [hunk 6]
```diff
@@ -0,0 +1,26 @@
...
+  const { productName } = product
+
+  if (!skuName || skuName === productName) {
+    return null
+  }
+
+  return (
+    <div
+      className={`${handles.skuNameContainer} flex items-start justify-center pv6`}
+    >
```
Comment (klzns) [inline react/ProductSummarySKUName.tsx:20]: People complain that our components styles are too opinionated. I would go to remove any default style in this case.
```suggestion
    <div className={handles.skuNameContainer}>
```

File: react/ProductSummarySKUName.tsx [hunk 7]
```diff
@@ -0,0 +1,26 @@
+import React from 'react'
+import { useCssHandles } from 'vtex.css-handles'
+import { useProductSummary } from 'vtex.product-summary-context/ProductSummaryContext'
+
+const CSS_HANDLES = ['skuNameContainer']
```
Comment (klzns) [inline react/ProductSummarySKUName.tsx:5]:
```suggestion
const CSS_HANDLES = ['skuNameContainer'] as const
```

File: docs/ProductSummarySKUName.md [hunk 8]
```diff
@@ -0,0 +1,58 @@
...
+
+:loudspeaker: **Disclaimer:** Don't fork this project; use, contribute, or open issue with your feature request.
+
+## Table of Contents
+- [Usage](#usage)
+  - [Blocks API](#blocks-api)
+  - [Configuration](#configuration)
+  - [Styles API](#styles-api)
+
+## Usage
```
Comment (klzns) [inline docs/ProductSummarySKUName.md:16]: This is the old README model. 

You can follow this instead:
https://github.com/vtex-apps/breadcrumb

**Review by klzns — CHANGES_REQUESTED** [review]
Thanks for the PR!

File: docs/ProductSummarySKUName.md (hunk 8 above)
Comment (wender) [reply docs/ProductSummarySKUName.md]: Done, please check

**Review by klzns — APPROVED** [review]
